tests/color/multiline_removal_suggestion.unicode.term.svg

```diff
@@ -57,7 +57,7 @@
 </tspan>
     <tspan x="10px" y="334px"><tspan class="fg-bright-blue bold">20</tspan><tspan> </tspan><tspan class="fg-bright-red">-                     )</tspan>
 </tspan>
-    <tspan x="10px" y="352px"><tspan class="fg-bright-blue bold">21</tspan><tspan> </tspan><tspan class="fg-bright-red">-                 })</tspan><tspan>.flatten()</tspan>
+    <tspan x="10px" y="352px"><tspan class="fg-bright-blue bold">21</tspan><tspan> </tspan><tspan class="fg-bright-red">-       </tspan><tspan>          }).flatten()</tspan>
```

Comment:
This is a regression; I suspect the cause is related to the tab characters in the test's `source`.